googleSheets.js

```diff
@@ -0,0 +1,66 @@
+const API_KEY = 'AIzaSyAJgXqtZgIPQ8r9pwFHAJrBxw0NNrztmhw';
```

Comment:
if there's a safer place to store this I'm open to moving it. not sure how you're deploying this project

Reply:
It's all just on our MacBooks... if that's what you mean? I honestly have no clue what I'm doing. 😂